articles/communication-services/how-tos/cte-calling-sdk/includes/manage-calls/manage-calls-web.md

```diff
@@ -328,7 +328,7 @@ const localVideoStreams = call.localVideoStreams;
 ```
 
 ## Manage chat thread
-Providing a chat ID is mandatory for making group calls and adding participants to existing calls. Associated chat and call have a separate list of participants. Before adding participants to the call, add the user to the chat to provide the best user experience and satisfy information barrier requirements. Adding a user to the call without adding the user to the chat can result in exceptions if an information barrier is set up.
+Providing a chat ID is optional for making group calls and adding participants to existing calls (We have a flag config to control if providing chat thread ID is mandatory or not). Associated chat and call have a separate list of participants. Before adding participants to the call, add the user to the chat to provide the best user experience and satisfy information barrier requirements. Adding a user to the call without adding the user to the chat can result in exceptions if an information barrier is set up.
```

Comment:
```suggestion
Providing a chat ID is optional for making group calls and adding participants to existing calls. Associated chat and call have a separate list of participants. Before adding participants to the call, add the user to the chat to provide the best user experience and satisfy information barrier requirements. Adding a user to the call without adding the user to the chat can result in exceptions if an information barrier is set up.
```

Reply:
> Can you clarify how the fact that there is flag config is relevant to developers?

The ECS config is set by our side which can control if developers can leverage the added APIs or not. That's why I mentioned it.